doc/docs_strategy.md

```diff
@@ -0,0 +1,56 @@
+# Documentation Strategy

```

Comment:
Not sure if we want this in the main branch?

Reply:
Can drop this if we feel its not needed

Reply:
I’ll remove it